docs/buildah-bud.md

```diff
@@ -41,6 +41,11 @@ Default certificates directory is _/etc/containers/certs.d_.
 
 Path to cgroups under which the cgroup for the container will be created. If the path is not absolute, the path is considered to be relative to the cgroups path of the init process. Cgroups will be created if they do not already exist.
 
+**--compress**
+
+This is legacy option, which has no effect on the image.  Buildah does not use a daemon. There
+is no reason to compress the input.
+
```

Comment:
I'm finding this confusing and the Docker documentation doesn't enlighten much.  By just reading this, I wouldn't know why the lack of a Buildah daemon is important to this parameter.  Would this hold?:

"This is a legacy option in Docker which no longer has an effect on an image.  As Buildah already compresses the image, there is no need to further compress it."

Reply:
I don't want to mention `Docker` at all.  As was explained to me, this flag compresses the imput data before sending it to the daemon.  If you were sitting on a MAC/Windows and had a big tar ball that you were doing a COPY or ADD, then this flag would cause the tar ball to the server, where it would be uncompressed.

Reply:
OK, would you be hip with my wording then but just drop "in Docker"?

Reply:
My suggestion:
```
This option is added to be aligned with other containers CLIs. 
Buildah doesn't communicate with a daemon or a remote server. 
Thus, compressing the data before sending it is irrelevant to Buildah.
```

Reply:
I like @ripcurld Suggestion. Do not want to add Docker to the buildah docs if we can avoid it.